src/Core/Models/Interfaces/IPermissions.cs

```diff
@@ -0,0 +1,15 @@
+namespace Bit.Core.Models.Interfaces 
+{
+    public interface IPermissions
+    {
+        public bool AccessBusinessPortal { get; set; }
```

Comment:
Remove `public` keywords from interface... all members (properties/methods) of an interface are already `public`, the compiler should be yelling at you here, or at least the optimizer/intellisense.

Reply:
Not applicable with IPermissions getting the boot